src/pages/guides/hybrid-plugins/getting-started/index.md

```diff
@@ -55,3 +55,54 @@ This is opposed to a standard UXP plugin’s method of accessing the file system
 let entry = await require('uxp').storage.localFileSystem.getFileForSaving("target.psd");
 document.saveAs.psd(entry);
 ```
+
+
+## Photoshop C++ SDK
+
+Photoshop offers a number of C based plugin APIs. These are documented in the [downloadable package available](https://console.adobe.io/servicesandapis).
```

Comment:
Did you mean downloadable package available "here"? It looks like you missed a word, somewhere.